pyincore/analyses/joplinempiricalbuildingrestoration/joplinempiricalbuildingrestoration.py

```diff
@@ -183,27 +183,27 @@ def get_spec(self):
                 {
```

Comment:
For this we can have the description as the one under the class declaration which looks like this:

"
Joplin Empirical Building Restoration Model generates a random realization for the restoration time of
    a building damaged in a tornado event to be restored to a certain functionality level. Functionality
    levels in this model are defined according to Koliou and van de Lindt (2020) and range from Functionality
    Level 4 (FL4, the lowest functionality) to Functionality Level 0 (FL0, full functionality).
"